core/README.md

```diff
@@ -2,4 +2,4 @@
 
 The main subpackage which implements basic building blocks (EOPatch, EOTask and EOWorkflow) and commonly used functionalities.
 
-For more information on the package content, visit [readthedocs](https://eo-learn.readthedocs.io/en/latest/eolearn.core.html).
+For more information on the package content, visit [readthedocs](https://eo-learn.readthedocs.io/en/latest/eotasks.html#core).
```

Comment:
should we have the links to the tasks? Or do we add a link to the root of the docs for the module, e.g. https://eo-learn.readthedocs.io/en/latest/reference/eolearn.core.html

Reply:
We could even do both. If you're reading on the docs you can conveniently jump to the codebase, and if you're on git you can conveniently jump to the relevant docs/tasks.

I can add this if you agree with the above.

Reply:
So it would be something along the lines of
```
For more information about the module visit [readthedocs](blabla.html), or see the [module task index](blaeotasks.html#core)
```

That sounds nice

Reply:
done